examples/CMakeLists.txt

```diff
@@ -1,10 +1,46 @@
-if($(AREG_BINARY) MATCHES "shared")
-    #These variables must be included in CMAKE_CXX_FLAGS
-	set(examples_CXXFLAGS "-DIMP_AREG_DLL")
-	set(examples_LDFLAGS  "Wl,-rpath=${AREG_OUTPUT_BIN} -L ${AREG_OUTPUT_BIN} -lareg ${LDFLAGS}")
-else()
-    #These variables must be included in CMAKE_CXX_FLAGS
-	set(examples_CXXFLAGS "-DIMP_AREG_LIB")
-	set(examples_LDFLAGS  "-L ${AREG_OUTPUT_LIB} -Wl,-Bstatic -lareg -Wl,-Bdynamic ${LDFLAGS}")
-endif()
 
+# Compile Options
+set(CompileOptShared "Wl,-rpath=${AREG_OUTPUT_BIN} -L ${AREG_OUTPUT_BIN}")
+set(CompileOptStatic "-L ${AREG_OUTPUT_LIB} -Wl,-Bstatic -Wl,-Bdynamic")
+
+# Add examples here in this list
+list(APPEND fileList
+	01_hello
+	02_buffer
+	03_file
+	04_trace
+	05_timer
+	06_threads
+	07_synch
+	08_service
+	09_svcmulti
+)
+
+# Iterating over fileList and extract each source file,
+# to create separate target and executable.
+foreach(item IN LISTS fileList)
+	unset(SRC)
+	set(itemDir ${AREG_EXAMPLES}/${item})
+	set(myDir ${itemDir}/src)
```

Comment:
Suggesting to rename myDir to (sub)ProjDir.

Reply:
`myDir` renamed to `subProjDir`.